api.md

```diff
@@ -887,12 +888,51 @@ SELECT detach_tablespaces('conditions');
 ```
 
 ---
+## distributed_exec() [](distributed_exec)
+
+This command is used on an access node to execute a SQL command across
+the data nodes of a distribute database. This command can be used to,
+e.g., create cluster-wide objects across all instances of a
+distributed database. A common use case is creating the roles and
+permissions needed in a distributed database.
+
+Note that the command is _not_ executed on the access node itself.
+
+#### Required Arguments [](distributed_exec-required-arguments)
+
+|Name|Description|
+|---|---|
+| `query` | The command to execute on data nodes. |
+
+#### Optional Arguments [](distributed_exec-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `node_list` | An array of data node names to execute the command across. Defaults to all data nodes if not specified. |
+
+#### Sample Usage [](distributed_exec-examples)
+
+Create the role `davide` across all data nodes in a distributed database:
+
+```sql
+SELECT distributed_exec('CREATE USER davide WITH PASSWORD ''jw8s0F4''');
```

Comment:
Nit, probably a matter of taste, but I like to use the dollar quoting syntax for this kind of stuff
```suggestion
SELECT distributed_exec($sql$CREATE USER davide WITH PASSWORD 'jw8s0F4'$sql$);
```

Reply:
cc @mtgentry useful for postgrescheatsheet.com?

Reply:
@feikesteenbergen Would you mind just `$$`?

Reply:
No, I don't mind just $$